src/status_im/multiaccounts/create/core.cljs

```diff
@@ -40,7 +41,7 @@
 
 (re-frame/reg-fx
  ::store-multiaccount
- (fn [[id key-uid hashed-password callback]]
+ (fn [[id key-uid hashed-password callback on-error]]
```

Comment:
can we rename callback to on-success so that we can c the `good pair` on-success/on-error? 😅